specification/azuredatatransfer/resource-manager/Microsoft.AzureDataTransfer/stable/2024-05-07/examples/createOrUpdateFlow.json

```diff
@@ -0,0 +1,36 @@
+{
+  "parameters": {
+    "connectionName": "testConnection",
+    "flowName": "testFlow",
+    "resourceGroupName": "testRG",
+    "api-version": "2024-05-07",
+    "subscriptionId": "00000000-0000-0000-0000-000000000000",
+    "flow": {
+      "properties": {
+        "storageAccountName": "testsa",
+        "storageContainerName": "testcontainer",
+        "flowType": "Blob",
```

Comment:
>"flowType": "Blob", [](http://example.com/codeflow?start=8&length=19)

in all examples I see that the flowType uses a value which does not exist. Maybe you can update the examples to include the new schema?

Reply:
Updated examples to include valid flow type.